Mathlib/Analysis/NormedSpace/ENormedSpace.lean

```diff
@@ -35,38 +35,42 @@ normed space, extended norm
 noncomputable section
 
 attribute [local instance] Classical.propDecidable
+set_option linter.deprecated false
```

Comment:
@fpvandoorn, why did you add this? I've removed it again in #24001 and everything seems fine.

Reply:
Oh, probably because I was considering deprecating the whole file. Thanks for the fix.